src/main/java/org/osc/manager/rest/server/api/DeviceApis.java

```diff
@@ -35,21 +35,24 @@
 import org.osc.manager.ism.api.IsmDeviceApi;
 import org.osc.manager.ism.entities.DeviceEntity;
 import org.osc.manager.ism.entities.DeviceMemberEntity;
-import org.slf4j.LoggerFactory;
-import org.osc.manager.rest.server.SecurityManagerServerRestConstants;
 import org.osc.sdk.manager.element.ManagerDeviceElement;
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.transaction.control.TransactionControl;
 import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 @Component(service = DeviceApis.class)
-@Path(SecurityManagerServerRestConstants.SERVER_API_PATH_PREFIX + "/devices")
+@Path("/devices")
```

Comment:
just to confirm the URL to access device api's would be IP:PORT/sm/devices right?

Reply:
@arvindn05 yes that is correct.